editor/import/resource_importer_scene.cpp

```diff
@@ -1240,7 +1240,7 @@ Error ResourceImporterScene::import(const String &p_source_file, const String &p
 
 	String root_type = p_options["nodes/root_type"];
 
-	if (scene->get_class() != root_type) {
+	if (root_type != "" && scene->get_class() != root_type) {
 		Node *base_node = Object::cast_to<Node>(ClassDB::instance(root_type));
```

Comment:
I think probably the only change in this whole file should be:
```
if (scene->get_class() != root_type) {
```
for
```
if (root_type != "Spatial") {
```
Since nothing can be "less" than Spatia. This way, if you specify something other than Spatial it's because you really want to override it with this type. If you don't, then you don't care what comes and using what is in the scene should be good..

Reply:
Why spatial though? I might want it to be a Node.. I don't have a use case in mind but I don't understand why taking away flexibility..?

Reply:
Which is the whole point of this PR